tests/ops/qubit/test_non_parametric_ops.py

```diff
@@ -918,16 +918,34 @@ def test_pauliz_general_power(self, n):
         assert op_pow[0].__class__ is qml.PhaseShift
         assert qml.math.allclose(op_pow[0].data[0], np.pi * (n % 2))
 
-    @pytest.mark.parametrize("n", (0.5, 2.5, -1.5))
-    def test_ISWAP_sqaure_root(self, n):
-        """Test that SISWAP is the square root of ISWAP."""
-        op = qml.ISWAP(wires=(0, 1))
-
-        assert op.pow(n)[0].__class__ is qml.SISWAP
+    @pytest.mark.parametrize(
```

Comment:
We can also think of whether or not we want to support mod-3 here

Reply:
This PR is just to fix a bug, not trying to add more features.